### ISSUE TITLE
Fix the document for norm op

### ISSUE THREAD

**Issue by emailweixu** [PaddlePaddle/Paddle#7266]
Here is the [doc](http://paddlepaddle.org/docs/develop/documentation/en/operators.html#permalink-97-norm)

**Comment by emailweixu** [comment]
And the name of this op is not very good because it is too general. "cross_channel_norm" might be better

**Comment by qingqing01** [comment]
Actually, I think a general norm operator is better and this general norm operator can support to do norm in any dimension. A long time ago, @guoshengCS had implemented a general norm operator,
 https://github.com/PaddlePaddle/Paddle/pull/4496 . This PR used `abs_op`, `reduce_sum_op` and `pow_op` to combine a norm operator in C++ code.  But when the `net_op` was removed, this `norm_op` is deleted.

And I think a general norm operator can support `cross_channel_norm_op`, when @sweetsky0901 plan to implement to `cross_channel_norm_op`, I discussed with her, we should implement a general norm operator.
